whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapQueryLimitTestIntegration.java

```diff
@@ -0,0 +1,261 @@
+package net.ripe.db.whois.api.rdap;
+
+import com.google.common.collect.Lists;
+import com.google.common.net.HttpHeaders;
+import net.ripe.db.whois.api.fulltextsearch.FullTextIndex;
+import net.ripe.db.whois.api.rdap.domain.Action;
+import net.ripe.db.whois.api.rdap.domain.Autnum;
+import net.ripe.db.whois.api.rdap.domain.Domain;
+import net.ripe.db.whois.api.rdap.domain.Entity;
+import net.ripe.db.whois.api.rdap.domain.Event;
+import net.ripe.db.whois.api.rdap.domain.Ip;
+import net.ripe.db.whois.api.rdap.domain.Link;
+import net.ripe.db.whois.api.rdap.domain.Nameserver;
+import net.ripe.db.whois.api.rdap.domain.Notice;
+import net.ripe.db.whois.api.rdap.domain.RdapObject;
+import net.ripe.db.whois.api.rdap.domain.Remark;
+import net.ripe.db.whois.api.rdap.domain.Role;
+import net.ripe.db.whois.api.rdap.domain.SearchResult;
+import net.ripe.db.whois.common.IntegrationTest;
+import net.ripe.db.whois.query.support.TestWhoisLog;
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.DirtiesContext;
+
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.HttpMethod;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.time.LocalDateTime;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.Matchers.*;
```

Comment:
Change your IDE settings to not use asterisk for importing packages (it's harder to see what's imported, and what changes)